### ISSUE TITLE
WebSocket: Error 500 when using adapter for $connect route

### ISSUE THREAD

**Issue by ceilfors** [laconiajs/laconia#49]
**Describe the bug**
WebSocket adapter returns 500 without any error. I couldn't find what is causing the error as the error message is swallowed for $connect route.

**To Reproduce**

serverless.yml
```yml
  manage-connection:
    handler: src/manage-connection.handler
    events:
      - websocket:
          route: $connect
```

handler code:
```js
const laconia = require("@laconia/core");
const adapterApi = require("@laconia/adapter-api");

const app = async () => {};
const webSocket = adapterApi.webSocket();

exports.handler = laconia(webSocket(app));
```

**Expected behavior**
1. Return 200 as $connect is a happy path
2. Error message should be surfaced to the user

**Actual behavior**
1. Return 500
2. Error message is swallowed, I needed to go to `createWebSocketAdapter.js` and put a console.log inside `catch` block

The error message:
```
2019-06-02T19:48:59.072Z	296c7fe1-bf76-44bb-a958-79f9351af52e	TypeError: First argument must be a string, Buffer, ArrayBuffer, Array, or array-like object.
at Function.Buffer.from (buffer.js:183:11)
at module.exports.event (/var/task/node_modules/@laconia/adapter-api/node_modules/@laconia/event/src/apigateway/getBody.js:2:10)
at module.exports.event (/var/task/node_modules/@laconia/adapter-api/node_modules/@laconia/event/src/apigateway/parseWebSocket.js:5:16)
at fromRaw (/var/task/node_modules/@laconia/adapter-api/node_modules/@laconia/event/src/apigateway/ApiGatewayWebSocketEvent.js:10:7)
at /var/task/node_modules/@laconia/adapter-api/src/createWebSocketAdapter.js:5:30
at laconia (/var/task/node_modules/@laconia/core/src/laconia.js:30:28)
at <anonymous>
```

**Comment by hugosenari** [comment]
The problem is that we are trying to buffer event.body (undefined).
I can't fix this because there are no documentation of expected behavior :)
Easier solution would be return null.

Maybe we could add other event information to ApiGatewayWebSocketEvent ie:
- queryStringParameters
- multiValueQueryStringParameters
- pathParameters

**Comment by ceilfors** [comment]
Thanks @hugosenari! 🎉 This is released under 1.10.0
